src/pages/SignUpPageNew/styles.js

```diff
@@ -0,0 +1,29 @@
+const styles = () => ({
+  root: {
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+  },
+  signUpCard: {
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    width: '536px',
+    marginTop: '37px',
+    marginBottom: '37px',
+    paddingBottom: '20px',
+  },
+  logo: {
+    width: '150px',
+    height: '107px',
+    marginTop: '28px',
+    marginBottom: '40px',
+  },
+  redirectToSignIn: {
+    marginTop: '20px',
+    color: '#5B5B5B',
+    textTransform: 'none',
+  },
+});
+
```

Comment:
again, dont see need to hardcode margin/paddings - let's meet about this sometime

Reply:
I could be missing something though - mainly just not sure about the hardcoded width (overflow small screens) and the margin/paddings

Reply:
The hardcoded width is for the major dropdown box messing everything up when a major name was longer than the actual box, tho it probably shouldn't matter now because all major names are shorter than the dropdown box now (as far as I can tell).

Reply:
I hardcoded the logo width and height because that's the true size of the logo. Honestly all the hardcoded margins and paddings are there so the whole page looks nice, since it's a little messed up if I take them out. I'm not sure how to balance things out nicely with <Card /> like how I can do with <Grid />, so I guess that's why there are more hardcoded margins and paddings here.